$OpenBSD: patch-Rakefile,v 1.4 2014/01/30 18:29:51 jasper Exp $

- Change various configuration variables to what they would be like when
  PuppetDB has been _installed_, as these variables are used in other scripts.
  The installer has been modified in tasks/install.rake
- Set OpenBSD's Ruby site dir.

--- Rakefile.orig	Mon Jan  6 22:48:56 2014
+++ Rakefile	Sun Jan 12 11:02:23 2014
@@ -57,19 +57,19 @@ if @pe
     @pe_version = ENV['PE_VER'] || '3.0'
     @java_bin = "/opt/puppet/bin/java"
 else
-    @install_dir = "/usr/share/puppetdb"
-    @etc_dir = "/etc/puppetdb"
-    @config_dir = "/etc/puppetdb/conf.d"
-    @lib_dir = "/var/lib/puppetdb"
+    @install_dir = "${PREFIX}/share/puppetdb"
+    @etc_dir = "${SYSCONFDIR}/puppetdb"
+    @config_dir = "${SYSCONFDIR}/puppetdb/conf.d"
+    @lib_dir = "${VARBASE}/db/puppetdb"
     @libexec_dir = case @osfamily
       when /redhat/, /suse/, /darwin/, /bsd/
         "/usr/libexec/puppetdb"
       else
         "/usr/lib/puppetdb"
       end
-    @link = "/usr/share/puppetdb"
+    @link = "share/puppetdb"
     @name = "puppetdb"
-    @sbin_dir = "/usr/sbin"
+    @sbin_dir = "sbin"
 end
 
 @initscriptname = "/etc/init.d/#{@name}"
@@ -94,7 +94,7 @@ case @osfamily
   when /suse/
     @plibdir = @pe ? PE_SITELIBDIR : (%x(ruby -rrbconfig -e "puts RbConfig::CONFIG['sitelibdir']").chomp)
   when /openbsd/
-    @plibdir = @pe ? PE_SITELIBDIR : '/usr/local/lib/ruby/site_ruby/1.9.1'
+    @plibdir = @pe ? PE_SITELIBDIR : '${MODRUBY_SITEDIR}'
 end
 
 @heap_dump_path = "#{@log_dir}/puppetdb-oom.hprof"
